setup.cfg

```diff
@@ -4,8 +4,20 @@ test=pytest
 [flake8]
 max-line-length = 120
 # W503 raises a warning when there is a line break before a binary operator.
-# This is best practice according to PEP 8 and the rule should be ignored. 
+# This is best practice according to PEP 8 and the rule should be ignored.
 #
 # https://www.flake8rules.com/rules/W503.html
 # https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
 ignore = W503
+
+[mypy]
+check_untyped_defs = true
+disallow_any_generics = true
+disallow_untyped_calls = true
+disallow_untyped_defs = true
+ignore_missing_imports = true
+no_implicit_optional = true
+warn_unused_ignores = true
+
+[mypy-tests.*,trino.auth,trino.client,trino.dbapi,trino.sqlalchemy.*]
+ignore_errors = true
```

Comment:
Why mypy errors are ignored it these packages? For instance I see that you addressed some in `trino.client.py`, is it leftover?

Reply:
@hovaesco I added [this](https://github.com/trinodb/trino-python-client/pull/208/files#r931114490) comment which hopefully explains why the change in `trino/client.py` is required.

Reply:
What is `mypy-tests.*` module? Some leftover?

Reply:
@hashhar `mypy-tests` is not a module, see [here](https://mypy.readthedocs.io/en/stable/config_file.html#config-file-format) for specifics, but per,

> Additional sections named [mypy-PATTERN1,PATTERN2,...] may be present, where PATTERN1, PATTERN2, etc., are comma-separated patterns of fully-qualified module names, with some components optionally replaced by the ‘*’ character (e.g. foo.bar, foo.bar.*, foo.*.baz). These sections specify additional flags that only apply to modules whose name matches at least one of the patterns.

the TL;DR is for the `tests.*`,  `trino.auth`, etc. modules errors are currently being ignored.